src/main/osd/osd.c

```diff
@@ -600,6 +605,26 @@ static void osdUpdateStats(void)
         stats.max_esc_rpm = rpm;
     }
 #endif
+
+    uint8_t throttleScaled = (constrain(rcData[THROTTLE], 1000, 2000) - 1000)*9/1000;
```

Comment:
```suggestion
    uint8_t throttleScaled = (constrain(rcData[THROTTLE], 1000, 2000) - 1000) * 9 / 1000;
```
Should we use `PWM_RANGE_MIN` and `PWM_RANGE_MAX`?

Reply:
Will do.